.github/workflows/cicd-release-validation.yml

```diff
@@ -0,0 +1,544 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This is the top-level workflow that runs on release branches.
+# It invokes other jobs to perform builds and run tests.
+# All jobs run in parallel, using build artifacts to synchronize jobs.
+#
+# If you add or remove any tests that use reusable-test.yml on a pull request,
+# you must update codecov.yml to match.
+
+name: CI/CD - Release validation
+
+on:
+  # Trigger the workflow when a release-tag is created, or a commit is pushed on a release branch.
+  push:
+    tags:
+      - 'v0.*' # To be updated on GA, and after that on every major release.
+    branches:
+      - 'release/*'
+
+  # Permit manual runs of the workflow.
+  workflow_dispatch:
+    inputs:
+      # Request a target release branch to run CI/CD on.
+      branch:
+        description: 'Target release branch to run CI/CD on.'
+        required: true
+        default: 'release/'
```

Comment:
nit: this is different from line 19 (- 'release/*')
does github correctly suggest a branch with a matching prefix?

Reply:
That should just be the default text GH writes into the field, so removing the "*" just makes it clear to fill in the version, if you don't go by just selecting the branch from the dropdown.